### ISSUE TITLE
Can I control the order of menu items? (lib-menu 2.0.0)

### ISSUE THREAD

**Issue by Hubro** [enonic/lib-menu#32]
I can't find any mention of the order of menu items in this library.

I have tried sorting my pages manually, by creation date, by publish date and by modified date, and the actual order of the menu items doesn't align with any of them. Manually sorting my pages in Content Studio has no effect on the main menu. Changing and publishing pages that are part of the main menu will move them around seemingly randomly.

As an additional curiosity, any time the Enonic database is dumped/loaded, the menu items are randomly shuffled.

I've had a look at the source:

```
var getChildMenuItems = function (parent) {
    return libs.content.query({
        count: 1000,
        filters: {
            boolean: {
                must: [
                    {
                        hasValue: {
                            field: "_parentPath",
                            values: [
                                "/content" + parent._path
                            ]
                        }
                    }
                    ,
                    {
                        hasValue: {
                            field: "x." + globals.appPath + ".menu-item.menuItem",
                            values: [
                                "true"
                            ]
                        }
                    }
                ]
            }
        }
    });
};
```

No sorting mechanism is included. Does that mean the order of menu items are practically random?

Is there any way to control the order of menu items?

**Comment by poi33** [comment]
This should have bin fixed in version 2.0.1.
It then uses the sortOrder of the parent to figure out how each menu should be sorted. 
Are you using 2.0.0 in your application?

**Comment by Hubro** [comment]
> This should have bin fixed in version 2.0.1.
> It then uses the sortOrder of the parent to figure out how each menu should be sorted.
> Are you using 2.0.0 in your application?

Ah I see. Yes, I'm using 2.0.0. I didn't know there was a newer version compatible with Enonic XP 6 :)

Thanks!
